grid_map_costmap_2d/test/CMakeLists.txt

```diff
@@ -3,14 +3,16 @@ ament_add_gtest(${PROJECT_NAME}-test
   test_costmap_2d_converter.cpp
 )
 
-ament_target_dependencies(${PROJECT_NAME}-test
-  ${dependencies}
+target_link_libraries(${PROJECT_NAME}-test
+  ${PROJECT_NAME}::${PROJECT_NAME}
 )
 
 ament_add_gtest(costmap-2d-ros-test
   test_costmap_2d_ros.cpp
 )
 
-ament_target_dependencies(costmap-2d-ros-test
-  ${dependencies}
+target_link_libraries(costmap-2d-ros-test
+  ${PROJECT_NAME}::${PROJECT_NAME}
 )
+
+
```

Comment:
[nits] Is this new line necessary?

Reply:
Thanks, fixed. 

I'm inclined to get pre-commit running in the repo for these sorts of linter nits that can be fixed automatically with a tool.